common/pigx-runner.in

```diff
@@ -539,7 +542,7 @@ else:
     print("Commencing snakemake run submission locally", flush=True, file=sys.stderr)
 
 if args.graph:
-    command.append("--dag")
+    command.append("--rulegraph")
```

Comment:
what's this about?  Did this command get renamed?  If so: when?
In Guix we're currently still building all the pipelines with Snakemake 5 (even though we also have versions 6 and 7).

Reply:
Sorry in 8 changed lines I managed to put something i didnt intend to... The rulegraph is the same as dag, only each rule gets used only once, so that it is still reasonably legible when using a lot of wildcards. But that wasnt supposed to be part of this PR, ill remove it once im back at my desk...